rdr_service/dao/questionnaire_response_dao.py

```diff
@@ -1018,6 +1018,7 @@ def _update_participant_summary(
                 participant_summary.remoteIdVerificationOrigin = participant_summary.participantOrigin
                 participant_summary.remoteIdVerificationStatus = False
                 participant_summary.remoteIdVerifiedOn = None
+            participant_summary.lastModified = datetime.now()
```

Comment:
I think lastModified should be in UTC, so maybe we can use clock.CLOCK.now()...that way we are also consistent with how lastModified is changed throughout this file 😃

Reply:
thanks!